emd_approx_relaxed.hh

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <cstdint>
+#include <algorithm>
+
+template <typename T>
+T emd_approx_relaxed (const T* w1, const T* w2, const T* dist, uint32_t size)
+{
+  static int idxes[1000*1000+7]; //to avoid any malloc in code
+  for (size_t i=0; i<size; i++)
```

Comment:
Can you please wrap all the single lines into {}? I know I didn't mention the coding standard, my mistake. It is [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html) with this "always braces" addition.